kazoo/protocol/connection.py

```diff
@@ -838,13 +824,9 @@ def _authenticate_with_sasl(self, host, timeout):
 
             try:
                 header, buffer, offset = self._read_header(timeout)
-            except ConnectionDropped:
+            except ConnectionDropped as ex:
                 # Zookeeper simply drops connections with failed authentication
-                six.reraise(
-                    AuthFailedError,
-                    AuthFailedError("Connection dropped in SASL"),
-                    sys.exc_info()[2],
-                )
+                raise AuthFailedError("Connection dropped in SASL") from ex
```

Comment:
This is a bit inconsistent with the refactoring above that adds a `% err` to the error messages, but here is omitted.

I suspect this is the right format and the others should be updated, but if not then this should be tweaked for consistency.

Reply:
[nit] I would also recommend you keep using `err` for exceptions, to stay consistent with the others.